SETUP.md

```diff
@@ -171,18 +171,30 @@ After setup, read about our [code styleguide](./STYLEGUIDE.md), our [test suites
 Note: Virtual Machine Users should check the [Alternative note](#alternative-use-an-ubuntu-vm) below before starting
 
 1. `sudo apt-get update`
-1. `sudo apt-get install -y git mysql-server mysql-client libmysqlclient-dev libxslt1-dev libssl-dev zlib1g-dev imagemagick libmagickcore-dev libmagickwand-dev openjdk-9-jre-headless libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev curl pdftk enscript libsqlite3-dev build-essential redis-server rbenv chromium-browser parallel`
```

Comment:
this wasn't available to me on a default Ubuntu 18 instance on EC2, and most developers don't need to run pegasus tests locally very often. however I'm happy to add it back for now, and maybe we can figure out what to do here once we get to the bottom of the pegasus test problem you're currently running into.

Reply:
I did have to update the dockerfile for pdftk, so maybe the instructions here should match what the dockerfile is doing, if the default apt-get isn't working
https://github.com/code-dot-org/code-dot-org/blob/fa1cdafb994273ba0044251c45d05659b5a10323/docker/dockerfiles/Dockerfile#L105

Reply:
Thanks @tim-dot-org ! would you expect the same command to work when run from the command line on your system, without editing the docker file? 
```
curl -sSL -o /tmp/pdftk-java_3.0.9-1_all.deb https://mirrors.kernel.org/ubuntu/pool/universe/p/pdftk-java/pdftk-java_3.0.9-1_all.deb
```
if so, I can add it as an option here.

Reply:
yeah! it works on mine (Ubuntu 20) (and docker ubuntu 18)

Reply:
whoa, I had no idea debian packages could be installed on ubuntu! this succeeds for me on Ubuntu 18.04 as well. unfortunately, it doesn't seem to eliminate the need for `pdftk`, and our pegasus tests still fail:
```
ubuntu@ip-10-0-0-21:~/src/cdo/pegasus$ rake test TEST=test/test_pdf_merger.rb

PDFMergerTest
  test_merge_local_from_file                                      FAIL (0.00s)
Minitest::Assertion:         Expected 'pdftk' to be installed.
```
I'm not sure where you found the installation step for pdftk-java, but if you can find a similar one for `pdftk` I'll include it here. 

In the longer term, we should probably change our pegasus code to use pdftk-java instead since it sounds like it is replacing pdftk, after which point it'll be easier for us to install necessary dependencies locally. I'm not going to tackle that right now, though.

Reply:
oh! didn't notice it was the java version. that makes sense, the other workarounds you have should be good for now